### DISCUSSION TITLE
nuclei ram consumption increases over time

### DISCUSSION THREAD

**Discussion by molitona** [projectdiscovery/nuclei#2528] [Q&A]
Hi, I've a nuclei template that will send 1 million post request with empty body request.
I've 4GB RAM and ram consumption increases overtime it went up to 3.8GB and my pc crashed of course ?

why that increase in ram usage ? can i limit it ?

**Comment by olearycrew** [comment]
I believe this is probably related to the 1 million requests, and would recommend trying on a computer with more RAM to see if that helps.
